# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ModuleLoadRequest.h
# Commit: 967e68ed5f56
# Full Hash: 967e68ed5f56f40a159da9664a1f789d32cff134
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 3
# Description:
#   Bug 1742437 - move DOM script context into own file and rename it; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138132
# ==============================================================================

diff -r 22649272ffae -r 967e68ed5f56 dom/script/ModuleLoadRequest.h
--- a/dom/script/ModuleLoadRequest.h	Fri Feb 18 12:45:51 2022 +0000
+++ b/dom/script/ModuleLoadRequest.h	Fri Feb 18 12:45:52 2022 +0000
@@ -46,7 +46,7 @@
 
   ModuleLoadRequest(nsIURI* aURI, ScriptFetchOptions* aFetchOptions,
                     const SRIMetadata& aIntegrity, nsIURI* aReferrer,
-                    DOMScriptLoadContext* aContext, bool aIsTopLevel,
+                    ScriptLoadContext* aContext, bool aIsTopLevel,
                     bool aIsDynamicImport, ModuleLoaderBase* aLoader,
                     VisitedURLSet* aVisitedSet, ModuleLoadRequest* aRootModule);
 